doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -139,22 +418,36 @@ Requirements:
 
    #. ``GML Simple Features``: ``SF-0``
 
-   .. image:: tewfs-1_1.jpg
+   .. image:: ./image/tewfs-1_1.png
 
 Run WMS 1.1 tests
 -----------------
 
-#. Start GeoServer with the ``citewms-1.1`` data directory. 
+#. Prepare the environment:
 
-#. Change directory back to the cite tools and run the tests::
+  - Start GeoServer with the citewms-1.1 data directory. Example:
 
-     ant wms-1.1
+   .. code-block:: shell
 
-   With the following parameters:     
+    cd <root of geoserver install>
+    export GEOSERVER_DATA_DIR=<root of geoserver sources>/data/citewms-1.1
```

Comment:
Still points to the old, non existing directory.

Reply:
solved in https://github.com/randomorder/geoserver/pull/3/commits/91da4cbdc686ba5397014ab7887381ee240f8a1b